protocols/discord.py

```diff
@@ -268,6 +283,37 @@ def _burst_new_client(self, guild, member, pylink_netobj):
         self._update_user_status(guild.id, uid, member.user.presence)
         return pylink_user
 
+    def _burst_webhook_agent(self, webhook_agent, guild, pylink_netobj):
```

Comment:
I wonder if it'd make sense to have this code in the DiscordServer class, and only call _burst_webhook_agent on a particular guild's network object when a webhook message is received there.

Reply:
Actually, let's worry about this later